### PR TITLE
React Clock - Version 0.0.1

### PR DESCRIPTION
**Pull request by mjbozzo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "dependencies": {
+        "webpack": "~1.8.2",
+        "react-tools": "~0.13.1",
+        "react": "~0.13.1",
+        "react-hot-loader": "~1.2.4",
+        "webpack-dev-server": "~1.8.0",
+        "grunt-webpack": "~1.0.8",
+        "matchdep": "~0.3.0",
+        "jsx-loader": "~0.13.1",
+        "react-mini-router": "~1.1.2",
```
Comment (amilcar-infante) [inline package.json]: We really need the react-mini-router dependency in this component?

File: src/scripts/react-clock.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        this.setState({
+            clockInterval: setInterval(this.getCurrentTime, 1000)
+        });
+    },
+    componentWillUnmount: function () {
+        clearInterval(this.state.clockInterval);
+    },
+    render: function () {
+        return (
+            <div className="unlock-screen-clock">
```
Comment (amilcar-infante) [inline src/scripts/react-clock.jsx]: This className should be a prop or should be removed

File: src/scripts/react-clock.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                <p className="current-time">
```
Comment (amilcar-infante) [inline src/scripts/react-clock.jsx:33]: This component should have a stylesheet?

File: package.json (hunk 1 above)
Comment (mjbozzo) [inline package.json]: removed

File: src/scripts/react-clock.jsx (hunk 2 above)
Comment (mjbozzo) [inline src/scripts/react-clock.jsx]: i will change it to react-clock

File: src/scripts/react-clock.jsx (hunk 3 above)
Comment (mjbozzo) [inline src/scripts/react-clock.jsx:33]: In this case i think not. There is not a stylesheet created for this

**Comment by amilcar-infante** [comment]
LGTM

**Comment by felipe-sanchez** [comment]
Minor comment. LGTM after that

**Pull request merged**